docs/resources/service.md

```diff
@@ -110,8 +110,6 @@ of the `name` attribute.
 
 * `namespace` - (Optional, Enterprise Only) The namespace to create the service within.
 
-* `partition` - (Optional, Enterprise Only) The partition the service is associated with.
-
```

Comment:
I'm not sure I understand why this is removed, isn't service partition a valid option?

Reply:
This was a mistake, thanks.